### PR TITLE
Add warning to getting started guide to carefully handle the .ova suffix

### PR DESCRIPTION
**Pull request by shivaani0505**
**What this PR does / why we need it**:
   Modify Getting started guide to add a warning for careful handling of .ova suffix in envvars.txt file when VM template is created using vSphere 

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes https://github.com/kubernetes-sigs/cluster-api-provider-vsphere/issues/671

**Special notes for your reviewer**:

_Please confirm that if this PR changes any image versions, then that's the sole change this PR makes._

**Release note**:

```release-note

```

### CODE REVIEW

**Comment by shivaani0505** [comment]
@yastij Could you give me needs-ok-to-test?

File: docs/getting_started.md [hunk 1]
```diff
@@ -50,6 +50,8 @@ It is required that machines provisioned by CAPV use one of the official CAPV ma
 
 [Create a VM template](https://docs.vmware.com/en/VMware-vSphere/6.7/com.vmware.vsphere.vm_admin.doc/GUID-17BEDA21-43F6-41F4-8FB2-E01D275FE9B4.html) using the OVA URL above. The rest of the guide will assume you named the VM template `ubuntu-1804-kube-v1.15.4`.
 
+**Note:** When creating OVA template via vSphere using URL method, make sure VM template name is same as value specified for environment variable VSPHERE_TEMPLATE in envvars.txt file, taking care of .ova suffix for the template name.
```
Comment (akutz) [inline docs/getting_started.md]: Hi @shivaani0505,

A few notes:

* Please use the appropriate articles:
  * _When creating OVA_ should be _When creating the OVA_
  * _using URL method_ should be _using the URL method_
  * _make sure VM template_ should be _make sure the VM template_
  * _name is same_ should be _name is the same_
  * _value specified_ should be _the value specified_
  * _specified for_ should be _specified by_
  * _environment variable_ should be _the environment variable_
  * _in envvars.txt file_ should be _in the `envvars.txt` file_
  * _taking care of .ova suffix_ should be _taking care of the `.ova` suffix_
* Please use proper formatting when appropriate. For example:
  * VSPHERE_TEMPLATE should be `VSPHERE_TEMPLATE`
  * envvars.txt should be `envvars.txt`
  * .ova should be `.ova`
* I like to add _please_ whenever it makes sense, such as _, make sure_ could become _, please make sure_

**Comment by akutz** [comment]
Hi @shivaani0505,

Thank you for this PR. The content looks great! I left some feedback on the syntax that I hope you have time to address. If you don't, I will remove the hold and merge as-is. However, I thought the notes were important to keep the content consistent with existing documentation.

**Comment by shivaani0505** [comment]
@akutz Thanks so much for your feedback. Good to know the guidelines for the doc. I will incorporate the feedback and re-submit.

**Comment by akutz** [comment]
/lgtm
/approve
/hold cancel

Thank you @shivaani0505, looks perfect!

**Pull request merged**